README.md

```diff
@@ -1,11 +1,40 @@
 # PMC
 
-PMC is a simple, yet flexible code injection framework for Pokémon White 2. It's powered by the `libRPM` dynamic loading library under the hood, reducing the process of code injection to no more than dragging and dropping a file. PMC will hook itself into the game's overlay loading routine during `GameInit()` and from there on will silently swap patch modules in and out as the parent process needs them.
+PMC is a simple, yet flexible code injection framework for Pokémon Black 2 and White 2. It's powered by the `libRPM` dynamic loading library under the hood, reducing the process of code injection to no more than dragging and dropping a file. PMC will hook itself into the game's overlay loading routine by placing executable code in the secure area, and executing it from `main()` via a hook. From there on, PMC will silently swap patch modules in and out as the parent process needs them.
+
+## Installation
+You can either snatch a binary build from the [releases](https://github.com/kingdom-of-ds-hacking/PMC/releases) page, or build it from source using the instructions below.
+
+Once you do that, open your project in `CTRMap-CE` (ensuring the `CTRMapV` plugin is installed), go to the `Extras` tab, and install PMC.
```

Comment:
May be desirable to add the version requirement for CTRMapV since older versions do not work with the universal releases. Numbers might be needless, how about just "latest version"?

Reply:
Sure, that works.